src/proof/InterpolationContext.h

```diff
@@ -23,14 +23,11 @@ class InterpolationContext {
     std::unique_ptr<ProofGraph> proof_graph;
 public:
     InterpolationContext(SMTConfig & c, Theory & th, TermMapper & termMapper, Proof const & t,
-                         PartitionManager& pmanager , int n = -1 );
+                         PartitionManager& pmanager, int n = -1 );
     ~InterpolationContext();
 
     void printProofDotty();
 
-    // Create interpolants with each A consisting of the specified partitions
-    void getInterpolants(const std::vector<vec<int> > & partitions, vec<PTRef> & interpolants);
-
```

Comment:
I need to check this method, maybe this was the one HiFrog was using.
In that case, I need to bring back its implementation; or design a different interface.